tailwind.config.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
         500: '#6B7280',
         700: '#374159',
         900: '#151926',
-        1000: '#7F88A7',
+        1000: '#656F95',
```

Comment:
Avem in assets un `search.png`, cred ca e unused. Putem sa-i dam delete si inchidem PR